src/content/decentralized-identity.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Decentralized Identity 
+description: An overview of decentralized identity on Ethereum
+lang: en
+template: use-cases
+emoji: ":id:"
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Centralized identity systems are fragmented, poorly secured, and offer minimal privacy protection
+summaryPoint2: Decentralized identity increases personal control of one's identity information.
+summaryPoint3: Decentralized ID projects built on Ethereum can revolutionize identity management. 
+---
+
+# What are identity, identifiers and attestations? {#what-are-identity-identifiers-attestations}
+
+## What is identity {#what-is-identity}
+
+Identity means an individual's sense of self, defined by a set of unique characteristics. Identity refers to the fact of being an _individual_, i.e., a distinct human entity. Identity could also refer to other non-human entities, such as an organization or authority.
+
+## What are identifiers? {#what-are-identifiers}
+
+An identifier is a piece of information that can be used to distinguish a person's identity. Common identifiers include the following: 
+- Name
+- Social security number/tax ID number 
+- Mobile number 
+- Date and place of birth
+- Digital identification credentials, e.g., passwords, email address, usernames
```

Comment:
```suggestion
- Digital identification credentials, e.g., email addresses, usernames, avatars
```

Reply:
(Wouldn't use passwords here... those are more in line with authentication than an identifier)